whats_fresh/whats_fresh_api/templates/list.html

```diff
@@ -9,14 +9,22 @@
 {% endblock header %}
 
 {% block title %}
-<h2 class='light'>{{ title }}</h2>
+<h2 class='light'><a id="form-title"href="/entry/{{list_url}}">{{ title }}</a></h2>
 <p class='light accent'>Click to edit or add a <span class="new"><a href='{{ new_url }}'>New One</a></span></p>
 {% endblock title %}
 
 {% block content %}
+<form class="search-bar" action="">
+<input id="search-list" type="text" name="search" {% if search_text %} value="{{ search_text }}" {% endif %} required="true">
+<button id="search-btn" type="submit">Search</button>
+</form><br>
 <div class='message'><h4 class='light accent'>{{ message }}</h4></div>
     <div class='list'>
 {% for item in item_list %}
+{% if item != None %}
+{% comment %}
```

Comment:
I would make this comment all on one line, and clarify that with Whoosh, sometimes items `in` items are None -- perhaps "Bad workaround for Whoosh searches containing None items"?